### PR TITLE
Fix/style fixes

### PR DESCRIPTION
**Pull request by themonster2015**
## Description

* Fix expanding latest topic widget on subscribe button hovering.
* Adjust dropdown styling for search and list apps
* Spacing in notif app.
## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

**Review by quininez — APPROVED** [review]

File: ui/apps/notifications/src/components/pages/customize-notification-page.tsx [hunk 1]
```diff
@@ -293,11 +293,11 @@ const CustomizeNotificationPage: React.FC<CustomizeNotificationPageProps> = ({
         padding={'py-2,px-0'}
         customStyle="h-full md:h-min space-y-4"
       >
-        <Text variant="h5" align="center">
+        <Text variant="h5" align="center" customStyle="pb-2">
           {initial ? t('Customize Your Notifications') : t('Notifications Settings')}
         </Text>
-        <Divider customStyle="my-2" />
-        <Stack direction="column" customStyle="mx-4">
+        <Divider customStyle="!mt-0" />
```
Comment (didd) [inline ui/apps/notifications/src/components/pages/customize-notification-page.tsx:299]: The `Divider` component has `apply` function used to overwrite styles. Did you encounter a situation where simply using `mt-0` isn't working for you ? Btw this comment applies for all cases you where you are using `! ` to override styles.

Comment (didd) [reply ui/apps/notifications/src/components/pages/customize-notification-page.tsx:299]: Haha the approval came in as I was typing this comment :)

Comment (quininez) [reply ui/apps/notifications/src/components/pages/customize-notification-page.tsx:299]: It's a valid question, I assumed the styles weren't applying correctly, though it is preferable to not use `!`

Comment (themonster2015) [reply ui/apps/notifications/src/components/pages/customize-notification-page.tsx:299]: Ah I see Didd :)). Yes in this case simply using mt-0 didn't work to help remove the margin.

Comment (didd) [reply ui/apps/notifications/src/components/pages/customize-notification-page.tsx:299]: > It's a valid question, I assumed the styles weren't applying correctly, though it is preferable to not use !


I agree. And I was also curious to understand the situation where simply using the twind class isn't working.

**Pull request merged**